pkgs/top-level/all-packages.nix

```diff
@@ -4050,6 +4050,8 @@ with pkgs;
 
   btrfs-snap = callPackage ../tools/filesystems/btrfs-snap { };
 
+  ssdfs-utils = callPackage ../tools/filesystems/ssdfs-utils { };
```

Comment:
Why not ssdfs-tools, like the repo (and also the alpine package)?

Reply:
Good catch. I also wasn't sure which of the two to use. I ended up going with what the upstream author seems to intend:
 - commit messages reference `ssdfs-utils`: https://github.com/dubeyko/ssdfs-tools/commits/master
 - the name of the autoconf project is also `ssdfs-utils`: https://github.com/dubeyko/ssdfs-tools/blob/master/configure.ac#L6